documentation/Installation.md

```diff
@@ -163,9 +163,9 @@ For cluster machines, ensure the following requirements are met:
 
 * The following distributives and versions are supported:
 
-  * Centos 7.5+, 8.4, 9
```

Comment:
Do we need to exclude only CentOS 7.5 version or all of the versions from 7.5 and higher?